CONTRIBUTING.md

```diff
@@ -80,9 +80,11 @@ There are two ways of running the `goat` locally:
     Note: Linter env variables must be formatted as <LINTER_CONFIG>, i.e. RUFF_CONFIG, CFN_LINT_CONFIG,
     etc., and the values supplied will overwrite the default arguments supplied in the goat.
 
-4. Autofix is available for certain linters and can be invoked in two ways:
-   1. `docker run -e INPUT_AUTO_FIX="true" -v $PWD:/goat/ -rm <hash>`
-   2. `pipenv run invoke reformat`
+4. Autofix is available for certain linters, is disabled by default when running in a pipeline,  
```

Comment:
I thought for a pipeline we wanted to have a default that:
1. Runs the linters.
2. If any of the linters failed and auto fix was enabled (the default), it runs autofix.
3. Re-run linting.  If it now succeeds, (1) print the linter errors, and after those (2) print a message indicating that the issues _could_ be autofixed using <insert the command the user should run to autofix, like `pipenv run invoke reformat`>.  If the second linting still fails, just print the linter errors

Reply:
From my Wrike task comment:
```
1. If you are in a pipline (CI is true) then print a message saying "This can be autofixed!"
2. If you are not in a pipeline (CI is not true) then say "We autofixed your $linter failures, check it out and commit if you agree"
```

Reply:
Does this all make sense?

To be more explicit, locally I was thinking the default would be:
1. Run the linters
2. If any linters failed and autofix is enabled (default), run autofix.  Make sure to print a warning: Saw issues, trying to autofix...
3. Re-run linting and if successful, print the "we autofixed your $linter failures, check it out and commit" message.  Exit non-zero because the user needs to take action (review logs and commit).
4. If the second linter fails, just print the logs and err out stating we tried to autofix but it wasn't successful; your local dir may be dirty with some partial fixes.

Reply:
I think I have all of the options captured.